src/components/Card/components/CardTitle.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react'
+
+export interface CardTitleProps {
+    header: string
+}
+
+const CardTitle: FC<CardTitleProps> = ({ header }): JSX.Element => {
+    return <h3 itemProp='name' dangerouslySetInnerHTML={{ __html: header }} />
```

Comment:
Now this is a new component, we should change dangerouslySetInnerHTML to children.

Same of description comment

Reply:
Fixed